packages/code-editor/src/utils/utils.ts

```diff
@@ -57,7 +57,13 @@ export const getPrismLanguage = (language: string): Language => {
  * applies package overrides to the default vsCode prism theme
  * @returns modified prism theme object
  */
-export function getOverridenTheme() {
-  prismTheme.plain.backgroundColor = theme.colors.text
+export function getOverridenTheme(transparent: boolean) {
+  if (!transparent) {
```

Comment:
hugely minor point: typically you put the positive test first, then the negative goes in the else. Some linters will recommend that.